include/utilities/format/bp1/BP1Writer.h

```diff
@@ -527,4 +282,6 @@ class BP1Writer : public BP1Base
 } // end namespace format
 } // end namespace adios
 
+#include "utilities/format/bp1/BP1Writer.tcc"
```

Comment:
One of the reasons to separate the template implementation is to also not have to always have it included in user code.  I don't believe any of the template implementations here need to be public facing.  We should be able to drop this from the .h file and only include it in the .cpp